examples/interpreter-demo/wait_block.js

```diff
@@ -13,25 +13,25 @@
  */
 
 Blockly.defineBlocksWithJsonArray([{
-  "type": "wait_seconds",
-  "message0": " wait %1 seconds",
-  "args0": [{
-    "type": "field_number",
-    "name": "SECONDS",
-    "min": 0,
-    "max": 600,
-    "value": 1,
+  'type': 'wait_seconds',
+  'message0': ' wait %1 seconds',
+  'args0': [{
+    'type': 'field_number',
+    'name': 'SECONDS',
+    'min': 0,
+    'max': 600,
+    'value': 1,
   }],
-  "previousStatement": null,
-  "nextStatement": null,
-  "colour": "%{BKY_LOOPS_HUE}",
+  'previousStatement': null,
+  'nextStatement': null,
+  'colour': '%{BKY_LOOPS_HUE}',
```

Comment:
The quotes on the keys are definitely not required here.

Reply:
We quote tons of stuff that doesn't need to be, all over the place :/ these were only updated to single quotes because i format my files on save